core/catalog-core-cachecommands/pom.xml

```diff
@@ -78,7 +78,6 @@
         <dependency>
             <groupId>org.geotools</groupId>
             <artifactId>gt-cql</artifactId>
```

Comment:
There are bugs with gt-cql and how it handles zulu time and time zones.  If you can, you should use version 13 or later to get those fixes.